app/src/sharedTest/java/org/oppia/android/app/databinding/MarginBindableAdaptersTest.kt

```diff
@@ -0,0 +1,363 @@
+      setLayoutMarginEnd(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+      )
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+    )
+    assertThat(textView.marginEnd.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+    )
+  }
+
+  @Config(qualifiers = "sw600dp-land")
+  @Test
+  fun testMarginBindableAdapters_ltrEnabled_landscapeTablet_marginStartAndEndForLtrIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_textview)
+      setLayoutMarginStart(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+      )
+      setLayoutMarginEnd(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+      )
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+    )
+    assertThat(textView.marginEnd.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+    )
+  }
+
+  @Config(qualifiers = "port")
+  @Test
+  fun testMarginBindableAdapters_rtlIsEnabled_port_marginStartAndMarginEndForRtlIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_textview)
+      setLayoutMarginStart(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+      )
+      setLayoutMarginEnd(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+      )
+      textView.layoutDirection = View.LAYOUT_DIRECTION_RTL
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginEnd.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+    )
+    assertThat(textView.marginStart.toFloat()).isEqualTo(
+      context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+    )
+  }
+
+  @Config(qualifiers = "land")
+  @Test
+  fun testMarginBindableAdapters_rtlIsEnabled_landscape_marginStartAndMarginEndForRtlIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_textview)
+      setLayoutMarginStart(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_start)
+      )
+      setLayoutMarginEnd(
+        textView,
+        context.resources.getDimension(R.dimen.content_item_exploration_view_margin_end)
+      )
```

Comment:
Could this bit be moved to a helper to simplify the tests? Ditto in other test files.

Reply:
Done here and to other test files too.